src/components/CharactersPage/CharactersTable/CharactersTable.js

```diff
@@ -1,17 +1,26 @@
-import { useSelector } from "react-redux"
 import { isFinite } from "lodash"
+import { useQuery } from "@apollo/client"
+import { useState } from "react"
 
+import { GET_ALL_CHARACTERS } from "@/graphql/characterQueries"
 import CustomTable from "@/components/CustomTable"
 import {
   StyledTableCell,
   NoValueCell,
   EditButtonCell,
 } from "../../CustomTable/Cells"
-import * as characterSelectors from "@/selectors/characterSelectors"
 import replaceIfNull from "@/utils/replaceIfNullUtils"
 
 const CharactersTable = ({ onRowClick }) => {
-  const characters = useSelector(characterSelectors.characters)
+  const [rowsPerPage, setRowsPerPage] = useState(10)
+
+  const { error, loading, data, fetchMore } = useQuery(GET_ALL_CHARACTERS, {
+    fetchPolicy: "network-only",
+    nextFetchPolicy: "cache-first",
+    variables: {
+      first: rowsPerPage,
+    },
+  })
```

Comment:
Can you move your data fetching to the `CharactersPage` component and pass the relevant data down